.editorconfig

```diff
@@ -1,15 +1,15 @@
-# EditorConfig is awesome: https://EditorConfig.org
-
-# top-most EditorConfig file
-root = true
-
-[*]
-end_of_line = lf
-insert_final_newline = true
-charset = utf-8
-trim_trailing_whitespace = true
-indent_style = space
-indent_size = 4
-
-[*.yml]
-indent_size = 2
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = false
+charset = utf-8
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 2

```

Comment:
I believe `indent_size` should be left to 4 by default. If 2 is better for most files, then react, typescript, javascript codes specifically should have 4 space indentation.